### PR TITLE
941 Reconnect websocket

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoWebsocketStatusBadge.tsx [hunk 1]
```diff
@@ -25,16 +37,25 @@ export const WfoWebsocketStatusBadge = () => {
...
-                        <WfoStatusDotIcon color={theme.colors.success} />
+                        <WfoBoltFill color={theme.colors.success} />
                     ) : (
-                        <WfoXCircleFill color={theme.colors.danger} />
+                        <WfoBoltSlashFill color="yellow" />
                     )
                 }
-                style={{ paddingLeft: '8px' }}
+                css={websocketConnected ? connectedStyle : disconnectedStyle}
+                onClick={
```
Comment (Georgi2704) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoWebsocketStatusBadge.tsx:49]: `const onClickHandler = () => {...}` ? would look better

**Review by Georgi2704 — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoWebsocketStatusBadge.tsx (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoWebsocketStatusBadge.tsx:49]: Good. FIxed

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoWebsocketStatusBadge.tsx [hunk 2]
```diff
@@ -25,16 +37,23 @@ export const WfoWebsocketStatusBadge = () => {
...
             <WfoHeaderBadge
-                color={theme.colors.ghost}
                 textColor={theme.colors.shadow}
                 iconType={() =>
                     websocketConnected ? (
-                        <WfoStatusDotIcon color={theme.colors.success} />
+                        <WfoBoltFill color={theme.colors.success} />
                     ) : (
-                        <WfoXCircleFill color={theme.colors.danger} />
+                        <WfoBoltSlashFill color="yellow" />
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoWebsocketStatusBadge.tsx:45]: Must: Try to stick with the colors in the theme. The color `warning` or `warningText` from the theme are quite yellow to me too.

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoWebsocketStatusBadge.tsx (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoWebsocketStatusBadge.tsx:45]: Fair enough. This is still under debate with @wouter1975 though so it's going to change

**Pull request merged**